src/service/arguments/cli/tools/plan.ts

```diff
@@ -0,0 +1,29 @@
+import { Command } from "commander";
+import { CommandConstructor } from "@bc/service/arguments/cli/command-constructor";
+import { CLIActionType, ToolType } from "@bc/domain/cli";
+import { BuildSubCommandFactory } from "@bc/service/arguments/cli/build/build-subcommand-factory";
+import Container from "typedi";
+import { InputService } from "@bc/service/inputs/input-service";
+import { FlowType } from "@bc/domain/inputs";
+
+/**
+ * Create command parser for project list tool
+ * @implements {CommandConstructor}
+ */
+export class PlanCommand implements CommandConstructor {
+  createCommand(): Command {
+    const program = new Command(ToolType.PLAN);
+    program
```

Comment:
sorry but I don't get how plan command is executed, even no definition-file? :thinking: 
can you please specify it on documentation (README)?

Reply:
sorry if I'm saying bullshits, but I don't see the options you mean here... can you please clarify? thanks

Reply:
No no of course!

I am reusing the `BuiidlSubCommandFactory` to add all of these options (see [line 24](https://github.com/kiegroup/github-action-build-chain/pull/447/files#diff-5234e833c7ac9790861982e41d7cd4a26caa7ed08006820bad86f9b8a13db8d3R24)). This factory already has all the options configured and we use this factory for the `build` command so I am simply reusing it.

Reply:
Now I see it, sorry, I was blind!